include/dmlc/thread_group.h

```diff
@@ -11,6 +11,8 @@
 #include <dmlc/logging.h>
 #include <string>
 #include <mutex>
+#include <utility>
+#include <memory>
```

Comment:
not used?

Reply:
from cpplint failure https://travis-ci.org/dmlc/dmlc-core/jobs/503196406

Reply:
this refers to a succeeded job?